open_instruct/ppo_vllm_thread_ray_gtrl.py

```diff
@@ -136,6 +149,9 @@ class Args:
     """Seed of the experiment"""
     run_name: Optional[str] = None
     """A unique name of this run"""
+    saved_tokenizer_type: Literal["original", "tokenizer_config"] = "tokenizer_config"
```

Comment:
whats the rationale for this arg? debugging?

Reply:
I think before we used to save the original tokenizer, but now it's apparent that we should save the tokenizer got from `tokenizer_config`. Let me modify it then.